src/workloads/workloads.csproj

```diff
@@ -120,7 +120,7 @@
     </ItemGroup>
     
     <MakeDir Directories="$(ArtifactsNonShippingPackagesDir)" />
-    <ZipDirectory Overwrite="true" DestinationFile="$(ArtifactsNonShippingPackagesDir)Workloads.VSDrop.%(SdkFeatureBand.Identity).zip" SourceDirectory="$(VisualStudioSetupInsertionPath)%(SdkFeatureBand.Identity)"/>
+    <ZipDirectory Overwrite="true" DestinationFile="$(ArtifactsNonShippingPackagesDir)Workloads.VSDrop.%(SdkFeatureBand.Identity).$(PackageVersion).zip" SourceDirectory="$(VisualStudioSetupInsertionPath)%(SdkFeatureBand.Identity)"/>
```

Comment:
Have you verified this is a valid property at this point?

Reply:
May be worth putting an error condition in above at least if it's empty.

Reply:
in my local build it's showing as 6.0.9 (and it shows as available during the msbuild evaluation) and from what I can tell, numerous other projects use it. I tried using `build -c`, but the number remained stable. Not sure where the 'servicing-xxxxx`  pieces come from

Reply:
They would be present in a non-shipping package's version number, or in non-stable build mode. Stable version numbers are fine here.